### PR TITLE
Removed temporal scoping and isPrivate from road flags

### PR DESCRIPTION
**Pull request by TheRBajaj**
This PR removes temporal scoping and `isPrivate` flag from `road.flags` property.
Issue: https://github.com/OvertureMaps/schema-wg/issues/144

### Testing
Tested by modifying examples and creating new counterexamples accordingly and running test script.

### Acceptance Criteria
1. Delete temporal scoping from the `flags` property.
2. Delete `isPrivate` flag from the `roadFlag` enumeration.
3. Update the documentation for the `flags` enumeration to explain that flags represent physical state. (documentation for `road.flags` property already relays this information)

### CODE REVIEW

**Review by vcschapp — CHANGES_REQUESTED** [review]
Thanks Rahul. Looks right, just a couple small changes requested, similar to the changes req'd on https://github.com/OvertureMaps/schema/pull/104:

1. [ ] Split counterexamples so each file tests only one thing.
2. [ ] Fix up `"id"` field in counterexample files.
3. [ ] Fix your IDE/editor to ensure you have newlines at EOF.
4. [ ] Use `extExpectedErrors` feature. I think @marinrelatic can help you with this if you aren't sure what to do.

File: counterexamples/transportation/segment/road/bad-road-flags-invalid-isPrivate-flag.json
Comment (vcschapp) [inline counterexamples/transportation/segment/road/bad-road-flags-invalid-isPrivate-flag.json:19]: Hmmmm. Is it possible to do a bit better than this? Is there a line item that specifically relates to `flags`?

File: counterexamples/transportation/segment/road/bad-road-invalid-temporal-scoping.json
Comment (vcschapp) [inline counterexamples/transportation/segment/road/bad-road-invalid-temporal-scoping.json:19]: Thx!

**Review by vcschapp — CHANGES_REQUESTED** [review]

File: counterexamples/transportation/segment/road/bad-road-flags-invalid-isPrivate-flag.json
Comment (TheRBajaj) [reply counterexamples/transportation/segment/road/bad-road-flags-invalid-isPrivate-flag.json:19]: I couldn't find anything related to flags as that was my first intention since Im not a fan of this generic error message too, but this the string that gets returned when the error occurs.

Comment (vcschapp) [reply counterexamples/transportation/segment/road/bad-road-flags-invalid-isPrivate-flag.json:19]: Aren't the seventh and eighth lines in your screenshot (one of the two) what you're looking for?

```
[I#/properties/road/flags/0] [S#$defs/propertyDefinitions/road/properties/flags/items/oneOf/0/$ref] doesn't validate with '$defs/propertyDefinitions/roadFlag'
    [I#/properties/road/flags/0] [S#$defs/propertyDefinitions/roadFlag/enum] value must be one of "isBridge", "isLink", "isTunnel", "isUnderConstruction"
```

One of the two should work.

Comment (TheRBajaj) [reply counterexamples/transportation/segment/road/bad-road-flags-invalid-isPrivate-flag.json]: Changed it to the second error.

**Review by vcschapp — APPROVED** [review]
```
                      \0/
                       |_
                       |/
      .-------.        |
      |_______|____________
     /====0=[]
  __/______/__\____
 jgs               `\~^~^~^~^~^~^~^~^~^
```

Thanks!

Can you please squash your commit history down to one commit with a good title, and we'll merge it once we have the other approvals!

**Review by brad-richardson — APPROVED** [review]

**Comment by vcschapp** [comment]
Merging as its just a bug fix.

**Pull request merged**
